cycles.json

```diff
@@ -173,5 +173,12 @@
         "position": 31,
         "rotated": false,
         "size": 1
+    },
+    {
+        "code": "borealis",
+        "name": "Borealis",
+        "position": 32,
+        "rotated": false,
+        "size": 1
```

Comment:
Size 3, as midnight Sun and the other one are coming out later.

Reply:
ah, thanks!